packages/core/src/controllers/relayer.ts

```diff
@@ -152,9 +162,6 @@ export class Relayer extends IRelayer {
   public async subscribe(topic: string, opts?: RelayerTypes.SubscribeOptions) {
     this.isInitialized();
     let id = this.subscriber.topicMap.get(topic)?.[0] || "";
-
-    if (id) return id;
-
```

Comment:
Why are we removing this logic?

Reply:
same reason as above, if `subscribe` is attempted, we should subscribe and not rely on past subscriptions